cli/src/main/kotlin/OrtMain.kt

```diff
@@ -78,19 +78,24 @@ data class GlobalOptions(
  * than [threshold], print an according note and throw a ProgramResult exception with [severeStatusCode].
  */
 fun concludeSeverityStats(counts: Map<Severity, Int>, threshold: Severity, severeStatusCode: Int) {
-    var hasSevereIssues = false
+    var severeIssueCount = 0
 
     fun getSeverityCount(severity: Severity) =
-        counts.getOrDefault(severity, 0).also { hasSevereIssues = it > 0 && severity >= threshold }
+        counts.getOrDefault(severity, 0).also {
+            severeIssueCount += (it > 0 && severity >= threshold).compareTo(false)
```

Comment:
This one is actually tricky. Can we be sure that the counter is always incremented by one? The JavaDoc of `Boolean.compareTo()` says that it returns

> zero if this object represents the same boolean value as the argument; a positive value if this object represents true and the argument represents false; and a negative value if this object represents false and the argument represents true

So "a positive value", but not necessarily 1.

Reply:
True, I've changed the implementation.